test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -505,22 +505,39 @@ export default class PerfCompare extends Component {
                 curTestCI = parsedVariantsTest.metrics[j].value.CI;
 
                 // Get the metric's units
+                //Parse benchmark Variant and verify if the infos there
+                //get BenchmarkRouter & Metric files from server
+                let benchmarkMetricRequest = await fetch( `/api/getBenchmarkParsingInfo`, {
```

Comment:
Same issue as mentioned in Tabular View. Better names to reflect the data stored in it.